test/unit/apk-signing-specs.js

```diff
@@ -0,0 +1,146 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import ADB from '../../lib/adb.js';
+import sinon from 'sinon';
+import { getJavaForOs } from '../../lib/helpers.js';
+import path from 'path';
+import * as teen_process from 'teen_process';
+import * as utils from '../../lib/utils.js';
+import { tempDir } from 'appium-support';
+
+chai.use(chaiAsPromised);
+
+const selendroidTestApp = path.resolve(__dirname, '..', '..', '..', 'test',
+                                       'selendroid-test-app.apk'),
+      java = getJavaForOs(),
+      helperJarPath = path.resolve(__dirname, '..', '..', '..', 'jars'),
+      keystorePath = path.resolve(__dirname, '..', '..', '..', 'test', 'appiumtest.keystore'),
+      keyAlias = 'appiumtest',
+      password = 'android',
+      selendroidTestAppPackage = 'io.selendroid.testapp';
+
+describe('signing', async () => {
+  let adb = new ADB(),
+      withExecMock = (fn) => {
```

Comment:
we can go ahead and have a `let` in front of `withExecMock` to save the indentation and because it's a function, not just a simple assignment.